sphinx/environment/__init__.py

```diff
@@ -561,9 +560,8 @@ def read_doc(self, docname, app=None):
         """Parse a file and add/update inventory entries for the doctree."""
         self.prepare_settings(docname)
 
-        docutilsconf = path.join(self.srcdir, 'docutils.conf')
```

Comment:
So far, Sphinx searches `docutils.conf` from srcdir. But I think confdif is much appropriate for this case.